api/src/test/resources/org/openmrs/api/include/ObsServiceTest-initial.xml

```diff
@@ -32,6 +32,7 @@
   <obs obs_id="17" person_id="2" concept_id="1" status="FINAL" obs_datetime="2006-02-10 00:00:00.0" location_id="1" obs_group_id="2" creator="1" date_created="2006-02-11 15:57:35.0" voided="false" uuid="b5499df2-b17c-4b39-88a6-44591c165369"/>
   <obs obs_id="18" person_id="2" concept_id="21" status="FINAL" obs_datetime="2009-01-01 00:00:00.0" location_id="1" obs_group_id="17" value_coded="7" creator="1" date_created="2006-02-17 15:57:35.0" voided="false" uuid="0ee1248e-08aa-4a2c-9f34-fb3875f605e3"/>
 
+  <encounter encounter_id="3" encounter_type="1" form_id="1" encounter_datetime="2001-01-01 00:00:00.0" patient_id="8" location_id="1" visit_id="1" creator="1" date_created="2005-01-01 00:00:00.0" date_changed="2006-01-01 00:00:00.0" voided="0" uuid="7fffd6b9-0970-4967-88c7-0b7b50f12bc6"/>
```

Comment:
so fixed these 
        1) saveEncounter_shouldNotOverwriteObsAndOrdersCreatorOrDateCreated()

        2) saveObs_shouldCopyTheFormNamespaceAndPathFieldInEditedObs()


but for saveObs_shouldCopyTheFormNamespaceAndPathFieldInEditedObs(),i added the above line in the data set, to get the test passing,though this means that this test passes when we have an existing encounter,according to my changes i thought that only obs without an encounter are ingonred,as per this line in the changes i made 	if (obs.getPerson() != null && obs.getEncounter() != null && obs.getEncounter().getPatient() != null) {, @dkayiwa  @ibacher  what could be your take on this

Reply:
so far fixed three and left with two failing tests on this ie   1) ObsServiceTest.saveObs_shouldCascadeUpdateToNewChildObsGroups:1563  and 
ObsServiceTest.saveObs_shouldVoidOnlyOldObsWhenAllObsEditedAndNewObsAdded cc @dkayiwa @ibacher